### ISSUE TITLE
Documentation about convolution padding computation is missing

### ISSUE THREAD

**Issue by ppwwyyxx** [tensorflow/tensorflow#46980]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/api_docs/python/tf/nn/convolution?version=nightly

## Description of issue (what needs changing):

The documentation refers to an invalid link, https://tensorflow.org/api_guides/python/nn#Convolution, and the reference to that link is recently deleted in this commit:  https://github.com/tensorflow/tensorflow/commit/271df9ce498f630b6c417087f18b32f2f7446a83 (cc @MarkDaoust ). 

However, I think **the correct action to take is to make an alternative to the invalid link (rather than deleting it)**.

When the link existed in the past, it contains the formula of how padding is computed for convolution, which is obviously a necessary piece of information for users to know and must be in the documentation.

As a reference, I found some [incomplete third-party website](https://tensorflow.juejin.im/api_guides/python/nn.html) that contain the old version of this now-missing page:

![2021-02-06_23-50](https://user-images.githubusercontent.com/1381301/107140355-37990f80-68d6-11eb-9f2f-6c06d3503102.png)

**Comment by Bhavay192** [comment]
Hey @jvishnuvardhan , is anyone working on this?? Can you assign this to me??

**Comment by MarkDaoust** [comment]
Done. Give it a shot.

Those docs are coming from:

1. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn_ops.py#L1138-L1161
2. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn_ops.py#L998-L1135

**Comment by Bhavay192** [comment]
Hey @MarkDaoust , thanks for replying. Soo do I only need to attach the link of the third-party website in the links to the docs provided by you?? Like just edit the docs with the given link?? Where should I provide the link to the website?? Any specified location?? Please reply thanks.

**Comment by MarkDaoust** [comment]
The api-reference pages on tensorflow.org are built from the source.
Those two links I posted take you to the source for the source for `tf.nn.convolution`:

If you edit this, and your PR gets merged:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn_ops.py#L998-L1135

It will show up on tensorflow.org is a few days.

**Comment by armenabnousi** [comment]
The link is 404'ing again
